editor/editor/mode/visual.js

```diff
@@ -20,11 +20,25 @@ function Blocks( { blocks, onChange } ) {
 	return (
 		<div className="editor-mode-visual">
 			<div>
-				{ blocks.map( ( block, index ) =>
-					<div key={ index }>
-						{ wp.blocks.getBlockSettings( block.blockType ).edit( block.attributes, onChangeBlock( index ) ) }
-					</div>
-				) }
```

Comment:
>If we fallback to a different blockType as suggested above, all the changes here are not needed anymore.

I kept some changes mostly for consideration of:

- What if a block decides it doesn't need to be editable, and only assigns a `save` implementation? Should we support this? With updated logic it will simply show the front-end UI within the editor.
- We currently allow a block to register itself with neither `edit` nor `save` implementations, which will throw if left unchecked